### DISCUSSION TITLE
Latest update only downloading from Profile API

### DISCUSSION THREAD

**Discussion by murphysez** [UltimaHoarder/UltimaScraper#771] [Q&A]
Just downloaded the latest version today. I can log in just fine, list my subscriptions, but after I select a profile and run a scrape of everything, I only see files in the Profile folder. The other folders (Posts, etc...)  are created, but there's nothing in them. Config file is below.

If it matters, I no longer have auth_id and auth_hash cookies.

"onlyfans": {
      "settings": {
        "auto_profile_choice": "1",
        "auto_scrape_names": true,
        "auto_choice": "a",
        "auto_scrape_apis": true,
        "jobs": {
          "scrape_names": true,
          "scrape_paid_content": true
        },
        "download_directories": [
          ".sites"
        ],
        "file_directory_format": "{site_name}\\{username}\\{api_type}\\{value}\\{media_type}",
        "filename_format": "{filename}.{ext}",
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}\\{username}\\Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklist_name": "",
        "webhook": true
      }

**Comment by GotPythonized** [comment]
Look in your app dir. Search for ".profiles"
OnlyFans
└── .profiles
    └── OnlyFans
        └── default <<-- auto_profile_choice
            ├── auth.json



Change "auto_profile_choice": "1", for the folder name "default" (change it with your dirname)

So.. "auto_profile_choice": "default",





Then, in auth.json change the username, add sess and user-agent and that's IT :)

auth": {
    "username": "yourusername",
    "auth_id": "12345678",
    "auth_hash": "",
    "auth_uniq_": "",
    "sess": "98h3rv52rsq5i4dd3674fbuo7v",
    "app_token": "33d57ade8c02dbc5a333db34749ae26a",
    "user_agent": "mozilla\/5.0 (x11; ubuntu; linux x86_64; rv:85.0) gecko\/20100101 firefox\/85.0",
    "support_2fa": false,
    "active": true
}
